docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -1,8 +1,7 @@
 global:
```

Comment:
Actually for this file only the name and description are fetched and not other key variables. But anyway, I'm not telling you to remove all though. So no issues here